doc/modules/mod_muc_light_commands.md

```diff
@@ -0,0 +1,49 @@
+# MongooseIM's light multi-user chat commands set
+
+## Purpose
+This is a set of commands, providing actions connected with multi-user chat (light) features.
```

Comment:
"connected with multi-user chat (light) features."
Is it for both MUC and MUC light? Or only MUC light?

Reply:
You're right, a bit ambiguous, I'll change that